src/bundle/Resources/public/js/scripts/helpers/tooltips.helper.js

```diff
@@ -159,11 +167,13 @@
             bootstrap.Tooltip.getOrCreateInstance(tooltipNode).hide();
         }
     };
-
-    observer.observe(doc.querySelector('body'), observerConfig);
+    const observe = (baseElement = doc.querySelector('body')) => {
```

Comment:
Maybe it would be good to have the same default value as in parse? 🤔 
```suggestion
    const observe = (baseElement = doc) => {
```